extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -16,15 +16,19 @@
 
 import '../amp-pinterest';
 
-
 describes.realWin('amp-pinterest', {
   amp: {
     runtimeOn: false,
     ampdoc: 'single',
+    mockFetch: true,
     extensions: ['amp-pinterest'],
   },
 }, env => {
 
+  function widgetURL(pinID) {
```

Comment:
@nainar I ended up making a commit here to fix the tests as I thought it would be easier to use PR comments and tell you what I ended up doing. Please see comments inline below. The way I debugged this is:
1- `.only` one of the tests
2- ran `gulp test` with `--watch` which keeps the runner window open
3- Used the Debug button in the test runner and then used Dev Tools to put break points.

Reply:
Aaah yeah the --watch part is what I was having trouble with. And now that I read the end to end guide. It is mentioned there. /facepalm. Thank you for jumping in and taking a look here!